tests/jballerina-integration-test/src/test/resources/http/src/httpservices/41_serialize_xml.bal

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/log;
+import ballerina/mime;
+
+listener http:Listener listenerEP = new(9250);
+http:Client clientEP = new("http://localhost:9250");
+
+@http:ServiceConfig {
+    basePath:"/serialize"
+}
+service serializer on listenerEP {
+
+    @http:ResourceConfig {
+        methods:["GET"],
+        path:"/xml"
+    }
+    resource function serializeXml (http:Caller caller, http:Request req) {
+        //Create an `xml` body part as a file upload.
+        mime:Entity xmlFilePart = new;
+        xmlFilePart.setContentDisposition(
+                       getContentDispositionForFormData("xml file part"));
+        xmlFilePart.setFileAsEntityBody("../../../src/test/resources/http/ComplexTestXmlSample.xml",
```

Comment:
Shouldn't we use the path api for this? This path might be machine dependant

Reply:
this is resolved internally by io APIs